### DISCUSSION TITLE
Failure of Jaeger installation on Openshift using Minishift and Helm

### DISCUSSION THREAD

**Discussion by gandhi209** [jaegertracing/jaeger#3148] [Q&A]
I was trying to install Jaeger on openshift using Minishift and Helm by using below commands

helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
  helm install jaeger jaegertracing/jaeger

When seeing the Openshift console it was showing an error at Jaeger Collector,Query UI
![image](https://user-images.githubusercontent.com/87645839/126147481-5279bfce-d328-4951-bf46-9e5664147934.png)

Errors shown

**at Cassandra** : /usr/local/bin/docker-entrypoint.sh: line 35: /etc/cassandra/cassandra.yaml: Permission denied

**at Jaeger collector and Query** 
 : 2021/07/19 10:36:57 maxprocs: Leaving GOMAXPROCS=2: CPU quota undefined
{"level":"info","ts":1626691017.82801,"caller":"flags/service.go:117","msg":"Mounting metrics handler on admin server","route":"/metrics"}
{"level":"info","ts":1626691017.828087,"caller":"flags/service.go:123","msg":"Mounting expvar handler on admin server","route":"/debug/vars"}
{"level":"info","ts":1626691017.8286872,"caller":"flags/admin.go:105","msg":"Mounting health check on admin server","route":"/"}
{"level":"info","ts":1626691017.8287969,"caller":"flags/admin.go:111","msg":"Starting admin HTTP server","http-addr":":14269"}
{"level":"info","ts":1626691017.8288596,"caller":"flags/admin.go:97","msg":"Admin server started","http.host-port":"[::]:14269","health-status":"unavailable"}
2021/07/19 10:36:57 gocql: dns error: lookup jaeger-cassandra on 172.30.0.2:53: server misbehaving
{"level":"fatal","ts":1626691017.9041083,"caller":"command-line-arguments/main.go:75","msg":"Failed to init storage factory","error":"gocql: unable to create session: failed to resolve any of the provided hostnames","stacktrace":"main.main.func1\n\tcommand-line-arguments/main.go:75\ngithub.com/spf13/cobra.(*Command).execute\n\tgithub.com/spf13/cobra@v0.0.7/command.go:838\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\tgithub.com/spf13/cobra@v0.0.7/command.go:943\ngithub.com/spf13/cobra.(*Command).Execute\n\tgithub.com/spf13/cobra@v0.0.7/command.go:883\nmain.main\n\tcommand-line-arguments/main.go:137\nruntime.main\n\truntime/proc.go:204"}

Below is the status of the components
![image](https://user-images.githubusercontent.com/87645839/126177919-a099b643-07e9-4432-bf54-c07a525c9ece.png)

**Comment by jpkrohling** [comment]
This would be a question for the helm-charts repo. Unfortunately, looks like I can't move this there.

**Reply by jpkrohling** [reply]
In time, it would probably be helpful if you told us the versions of the software you are using (minishift, okd/openshift, ...)

**Reply by gandhi209** [reply]
Thanks @jpkrohling for the reply
minishift version : minishift v1.34.3+4b58f89

Okd/openshift : v3.11.0+ea42280

Please let me know the link where I can post this issue

**Reply by jpkrohling** [reply]
OpenShift v3.11 is really old and not something we actively test on. Please, use a recent version of OpenShift, like 4.6 or 4.7.
